src/main/java/de/retest/recheck/ui/diff/RootElementDifference.java

```diff
@@ -163,4 +155,8 @@ public int getCheckedUiComponentCount() {
 	public String getTitle() {
 		return title;
 	}
+
+	public RootElement getElement() {
```

Comment:
I suggest that we either call this method `getDescriptor()` (then it would correspond to the other methods), or even better: refactor the term "descriptor" to "element" in this class.

Reply:
Ok, renamed the other methods instead.